app/code/Transoft/Blog/Controller/Adminhtml/Post/Save.php

```diff
@@ -0,0 +1,88 @@
+<?php
+declare(strict_types=1);
+
+namespace Transoft\Blog\Controller\Adminhtml\Post;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\App\Action\HttpPostActionInterface;
+use Magento\Framework\App\Request\DataPersistorInterface;
+use Transoft\Blog\Model\ResourceModel\Post\CollectionFactory as PostCollectionFactory;
+
+/**
+ * Post save controller
+ */
+class Save extends Action implements HttpPostActionInterface
+{
+    const ADMIN_RESOURCE = 'Transoft_Blog::post';
```

Comment:
or **Transoft_Blog::blog_manage_items**. Refer to _isAllowed method.

Reply:
fixed